docs/embedding/sdk/appearance.md

```diff
@@ -114,7 +114,43 @@ const theme = defineMetabaseTheme({
     // Question
     question: {
       // Background color for all questions
-      backgroundColor: "#2D2D30",
+      backgroundColor: "#2E353B",
+
+      // Toolbar of the default interactive question layout
+      toolbar: {
+        backgroundColor: "#F3F5F7",
+      },
+
+      // <InteractiveQuestion.ChartTypeSelector /> component
+      chartTypeSelector: {
+        backgroundColor: "#fff",
+      },
+
+      // <InteractiveQuestion.QuestionSettingsDropdown /> component's button
+      questionSettingsButton: {
+        backgroundColor: "#fff",
+      },
+
+      // <InteractiveQuestion.EditorButton /> component
+      editorButton: {
+        borderColor: "#2E353B",
+
+        // Inactive states
+        iconColor: "#2E353B",
+        backgroundColor: "#F3F5F7",
+
+        // Active states
+        activeIconColor: "#F3F5F7",
+        activeBackgroundColor: "#2E353B",
+      },
```

Comment:
See [this Figma specs](https://www.figma.com/design/OamRqg07JOoC4bUqfCkh4t/SDK-Reference-APP?node-id=2799-3828&t=qKbOMdv7BxlCaSyX-1) for why these theme options are needed. Note that all is optional and has good defaults. In fact, everything in this PR has good defaults and none are required.

Reply:
I'm not a fan of exposing a ton of color. Can this button be themed with existing colors?

![image](https://github.com/user-attachments/assets/a92d60d5-fd20-4746-a9de-73e38158fb51)